app/core/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
```

Comment:
I see a misconception here: These models should go into the `recipe` application not in the `core` one. The idea of applications is that they encapsulate features and functionalities. so it makes little sense to put all the models inside a core application if you have a `recipe` one.

Reply:
Fully agree, removed core application, models have been moved to recipes application to keep it self-contained and avoid breaking encapsulation.